client/src/pages/models/train/[name].tsx

```diff
@@ -0,0 +1,22 @@
+import Prose from 'components/Prose';
+import {useRouter} from 'next/router';
+import React from 'react';
+
+export default function ModelTrainingView() {
+  const router = useRouter();
+  const {name} = router.query;
+
+  if (!name) {
+    // TODO make prettier
+    return <p>No model name provided.</p>;
```

Comment:
This is more of a question I have rather than a suggestion to change something - but, how would this branch ever be reached? For this code to be run, wouldn't the user have to navigate to `model/train/SOME_NAME` - and `SOME_NAME` can't be empty right?

Reply:
Removed in latest revision.